src/commonControls/numberControl/NumberControl.ts

```diff
@@ -499,39 +492,124 @@ export class NumberControl extends Control implements InteractionModelContributo
                         value: act.payload.renderedValue,
                     });
                 },
+                suggestValue: (act) =>
+                    i18next.t('NUMBER_CONTROL_DEFAULT_REPROMPT_SUGGEST_VALUE', {
+                        value: act.payload.renderedValue,
+                    }),
             },
             validation: [],
             confirmationRequired: false,
-            ambiguousPairs: [
-                [13, 30],
-                [14, 40],
-                [15, 50],
-                [16, 60],
-                [17, 70],
-                [18, 80],
-                [19, 90],
-            ], // TODO: generalize the default. in english it should suggest (113 <-> 130) etc.
             required: true,
-            apl: NumberControlAPLPropsBuiltIns.Default,
+            apl: {
+                enabled: true,
+                validationFailedMessage: NumberControl.defaultValidationFailureText(),
+                requestValue: NumberControlAPLPropsBuiltIns.defaultSelectValueAPLContent({
+                    validationFailedMessage: props.apl?.validationFailedMessage,
+                }),
+                renderComponent: NumberControlAPLComponentBuiltIns.ModalKeyPadRender.default,
+            },
             inputHandling: {
                 customHandlingFuncs: [],
             },
+            mostLikelyMisunderstandings: (value: number) => {
```

Comment:
this function should be a public exported function, as for standard APL.  mention the FQN of the default function in the tsDoc comment for the prop

Reply:
Updated